scripts/wave-lane-watch.ps1

```
@@ -0,0 +1,315 @@
+  if ($originExists) {
+    try {
+      $cnt = (& git -C $repoRoot rev-list --count "origin/main..origin/$branch" 2>$null)
+      if ($cnt) { $ahead = [int]($cnt.Trim()) }
+    } catch {}
+  }
+
+  $lastPushAge = $null
+  if ($originExists) {
+    $iso = (& git -C $repoRoot log -1 --format=%cI "origin/$branch" 2>$null)
+    if ($iso) {
+      try { $lastPushAge = [math]::Round(((Get-Date) - [datetime]$iso).TotalMinutes, 1) } catch {}
```

PSScriptAnalyzer (1.25.0)

[warning] Empty catch block is used. Please use Write-Error or throw statements in catch blocks.

(PSAvoidUsingEmptyCatchBlock)